config/config.go

```diff
@@ -43,8 +44,11 @@ type OptionalConfigParams struct {
 }
 
 var (
-	errCameraMustHaveName    = errors.New("\"camera[name]\" is required")
-	errSensorsMustNotBeEmpty = errors.New("\"sensors\" must not be empty")
+	errCameraMustHaveName              = errors.New("\"camera[name]\" is required")
+	errSensorsMustNotBeEmpty           = errors.New("\"sensors\" must not be empty")
+	errLocalizationInOfflineModeSuffix = "= 0, enable_mapping = false, and existing_map provided. localization in offline mode not supported."
+	errLocalizationInOfflineMode       = newError(fmt.Sprintf("data_rate_msec %s", errLocalizationInOfflineModeSuffix))
+	errLocalizationInOfflineModeIMU    = newError(fmt.Sprintf("camera[data_freq_hz] %s", errLocalizationInOfflineModeSuffix))
```

Comment:
[optional] if we do need two messages, suggestion to just write the full message out twice for easier readability

Reply:
This caused a lint error for too many characters in the line